### PR TITLE
Add new migration doc structure

### PR DESCRIPTION
**Pull request by pfmackin**
Add new migration doc structure.  This will be used to fill in details in subsequent PRs.

### CODE REVIEW

File: content/en/docs/guides/migrate/install/_index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Installation"
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply content/en/docs/guides/migrate/install/_index.md:2]: They are no longer Verrazzano components.  They are OCNE components that used to run on Verrazzano.

File: content/en/docs/guides/migrate/install/_index.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+description: "Learn how to install components on OCNE 2.0"
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply content/en/docs/guides/migrate/install/_index.md:3]: will do

File: content/en/docs/guides/migrate/install/cert-manager/_index.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+title: "cert-manager"
+description: "Learn how to install cert-manager on OCNE 2.0"
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply content/en/docs/guides/migrate/install/cert-manager/_index.md:3]: will do

File: content/en/docs/guides/migrate/integrate/_index.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+---
+title: "Integration"
+description: "Learn how to integrate OCNE 2.0 components with each other"
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply content/en/docs/guides/migrate/integrate/_index.md:3]: will do

File: content/en/docs/guides/migrate/integrate/cert-manager/_index.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+---
+title: "cert-manager"
+description: "Learn how to integrate cert-manager with other components"
```
[the remark this answers is not in the document]
Comment (pfmackin) [reply content/en/docs/guides/migrate/integrate/cert-manager/_index.md:3]: ok

**Comment by pfmackin** [comment]
> that to migrate Verrazzano components to OCNE, you need to first install them in OCNE and then integrate them with other OCNE components

Ok will do

**Review by rosemarymarano — APPROVED** [review]

**Pull request merged**